setup.cfg

```diff
@@ -27,9 +27,11 @@ exclude = build/*.py,docs/*.py,*/.ropeproject/*
 ignore = E128
 max-line-length = 99
 
+[check-manifest]
+ignore = hamster_gtk/resources/hamster-gtk.gresource
```

Comment:
Because otherwise ``check-manifest`` will notice that ``hamster-gtk.gresource`` is part of the sdist but not of the repository and hence will fail. As this is exactly what we want however, this line is needed.